docs/reference/alert-schema.asciidoc

```diff
@@ -142,4 +142,19 @@ Type: flattened
 |N/A | `kibana.alert.rule.producer` | Type: keyword
 |N/A | `kibana.alert.rule.rule_type_id` | Type: keyword
 
+|N/A | `kibana.alert.suppression.terms.field` | The field(s) used to group alerts for suppression.
```

Comment:
```suggestion
|N/A | `kibana.alert.suppression.terms.field` | The fields used to group alerts for suppression.
```

Reply:
I'm pretty sure we don't use the parenthesis "s" in our docs so I changed it to plural.